BotRoyaleUnity/Assets/Scripts/Networking/SocketConnectionHandler.cs

```diff
@@ -144,5 +155,9 @@ public void OnSocketEvent(string eventName, Action<SocketIOEvent> onEventAction)
         socket.On(eventName, onEventAction);
     }
 
+    public SocketIOController GetSocket(){
```

Comment:
Ideally we dont need to expose the underlying socket library at all, instead providing public methods that do whatever socket related things we need in this class